Software/bsw/static/COM/CanIf/inc/CanIf_Types.h

```diff
@@ -429,7 +417,7 @@ typedef struct
 				L-SDU for reading its notification status.
 				True: Enabled False: Disabled
 				dependency: CANIF_READTXPDU_NOTIFY_STATUS_API must be enabled.*/
-				#if(CANIF_PUBLIC_READ_TX_PDU_NOTIFY_STATUS_API==STD_ON)
+				#if(CANIF_PUBLIC_READTXPDU_NOTIFY_STATUS_API==STD_ON)
```

Comment:
please use the cfg names as they are as Macro's names are in UpperCase separated by a "_" character.